protoc-gen-prost-crate/Cargo.toml

```diff
@@ -16,7 +16,7 @@ categories = ["encoding"]
 
 [dependencies]
 once_cell = "1.10.0"
-prost-build = { version = ">=0.11.0, <0.11.2", default-features = false }
+prost-build = { version = "0.11.4", default-features = false }
```

Comment:
I needed to update `prost-build` everywhere to get crate resolution to work. Is there any reason this was locked? is this a problem?

Reply:
This is fine. It was locked to specifically _not_ 0.11.3 as that introduced an unexpected `println!`, which would interfere with our communication with `protoc`. Because many people do a cargo install without `--locked`, they were pulling in that version and we had reports of a breakage. Now that there is a version beyond that, we can again semver bind to it.